From a842dba6b0535a1264cb101f113e5da1e418f7d1 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Mon, 22 Apr 2024 14:08:55 +0100
Subject: [PATCH] Update autosuspend hwdb

(cherry picked from commit 56095ad6c0ba7441649ddc2fd075cea3040702ac)

Related: RHEL-5950
---
 hwdb.d/60-autosuspend-fingerprint-reader.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
index 8697c460ab..dc474c5e7c 100644
--- a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
+++ b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
@@ -189,6 +189,7 @@ usb:v27C6p5840*
 usb:v27C6p6014*
 usb:v27C6p6092*
 usb:v27C6p6094*
+usb:v27C6p609A*
 usb:v27C6p609C*
 usb:v27C6p60A2*
 usb:v27C6p60A4*
